### PR TITLE
Add NetEaseCrowd dataset

### PR DESCRIPTION
**Pull request by shenxiangzhuang**
## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

## Dataset info
Adding our open-source dataset, NetEaseCrowd(https://github.com/fuxiAIlab/NetEaseCrowd-Dataset).

>NetEaseCrowd is a large-scale dataset for long-term and online crowdsourcing truth inference, which contains about 2,400 workers, 1,000,000 tasks, and 6,000,000 annotations collected over 6 months. We believe that this dataset could be an invaluable asset to the Toloka/crowd-kit community by providing a new benchmark for crowdsourcing-related research and development.

### CODE REVIEW

File: crowdkit/datasets/_loaders.py [hunk 1]
```diff
@@ -249,6 +281,15 @@ def load_dataframes(data_path: str) -> Tuple[pd.DataFrame, "pd.Series[Any]"]:
...
     },
+    "netease_crowd": {
+        "loader": load_netease_crowd,
+        "description": "NetEaseCrowd(https://github.com/fuxiAIlab/NetEaseCrowd-Dataset)"
+        "is a large-scale crowdsourcing annotation dataset based on a mature Chinese data "
+        "crowdsourcing platform of NetEase Inc.. NetEaseCrowd dataset contains about 2,400 workers, "
+        "1,000,000 tasks, and 6,000,000 annotations between them, "
+        "where the annotations are collected in about 6 months."
+        "This dataset is licensed under CC BY-SA 4.0",
+    },
```
Comment (dustalov) [inline crowdkit/datasets/_loaders.py:292]:
```suggestion
    "netease_crowd": {
        "loader": load_netease_crowd,
        "description": "NetEaseCrowd(https://github.com/fuxiAIlab/NetEaseCrowd-Dataset) "
        "is a large-scale crowdsourcing annotation dataset based on a mature Chinese data "
        "crowdsourcing platform of NetEase Inc.. NetEaseCrowd dataset contains about 2,400 workers, "
        "1,000,000 tasks, and 6,000,000 annotations between them, "
        "where the annotations are collected in about 6 months. "
        "This dataset is licensed under CC BY-SA 4.0",
    },
```

**Review by dustalov — APPROVED** [review]
Thank you for a very well-done PR! I noticed a small imperfection in the dataset metadata. Could you please check my suggestion?

**Comment by shenxiangzhuang** [comment]
> Thank you for a very well-done PR! I noticed a small imperfection in the dataset metadata. Could you please check my suggestion?

Thanks a lot for your carefully review!

**Comment by dustalov** [comment]
Great job, thank you again!

**Pull request merged**